chore(readme): 리드미 업데이트 -1

imella0707 opened the pull request:
리드미 업데이트했습니다. 
branch 이름을 91-issue 로 정했습니다. 
수정할 내역 있으면 알려주세요 !

imella0707 committed:
chore(readme): 리드미 업데이트 -1

리드미 업데이트했습니다. 
수정할 내역 있으면 알려주세요 ~!

imella0707 committed:
chore(readme): 리드미 업데이트 -1 (#93)

chore(readme): 리드미 업데이트 -1 (#93)

minhyeok0328 commented on chore(readme): 파이프라인 이미지 업데이트-2:
이거 파일명을 README.md로 바꿔야 repository 방문할 때 보입니다!
파일명이 잘못되었네요

imella0707 replied:
아 어쩐지 안보여서, 다시바꾸겠습니다 ! 일단 내용을 좀 써봤는데 수정할 부분 있으면 알려주세용 ~

minhyeok0328 commented on README.md:
```diff
@@ -1 +1,198 @@
+## 🖥️ 개발 환경 설정
+
+- **GCP 인스턴스 사용**
+  - 리전: 서울 (asia-northeast3-a) 
```
asia-northeast3-a
뒤에 -a 빼야 합니당
a 영역만 쓰는게 아니니까여

imella0707 replied:
오 네넵 삭제하겠습니다.

imella0707 committed:
chore(readme): 리드미 업데이트 -2 (#93)

리드미 파일의 이름을 다시 수정해보았습니다.

minhyeok0328 commented on README.md:
```diff
@@ -1 +1,198 @@
+![image](https://github.com/user-attachments/assets/8da81757-4720-4548-bd5f-f1b66901de49)
+
+
+- 데이터 파이프라인은 4개의 주요 계층으로 구성되어 있음.<br>
```
계층 > 단계로 단어 바꿔야 할 것 같아요
계층은 적절하지는 않은 단어 같습니다

imella0707 replied:
저도 단계가 더 적절한 단어인 거 같습니다. 다시 보니까 어색한 게 보이네요 !

minhyeok0328 commented on README.md:
```diff
@@ -1 +1,198 @@
+#### 1) Storage Layer (입력 저장소 계층)
+- 데이터 소스:
+  - CSV 파일 업로드를 통해 데이터를 입력받음.
+  - GCS(Google Cloud Storage)에 직접 데이터를 저장하거나, GCS에서 데이터 가져오는 것도 가능.
```
`GCS(Google Cloud Storage)에 직접 데이터를 저장하거나, GCS에서 데이터 가져오는 것도 가능.`
이게 저희 파이프라인 역할 및 각 단계에서 어떻게 동작하는지 설명하는데
데이터 가져오는 것도 가능이라고 쓰면 GCS 기능 설명하는 것 같아서 조금 어색해 보입니당

imella0707 replied:
넵. 'GCS에서 데이터 가져오는 것도 가능' 이 부분 삭제하고,  다시 보니, '저장소' → '저장'으로 수정하고 싶어서 일단 그렇게 업데이트 해보겠습니다 ~!

imella0707 committed:
chore(readme): 리드미 업데이트 -3 (#93)

1.리전 영역 수정
2.Storage 한국어 번역할 때 저장소 → '저장'으로 단어 변경
3.GCS 기능 설명 삭제 :'GCS에서 데이터 가져오는 것도 가능' 부분

dnlpys commented on README.md:
```diff
@@ -1 +1,198 @@
+      - 이를 통해 이전 모델로 롤백하거나 다양한 모델을 비교할 수 있음.
+  
+    - 모델 상태 관리: 모델의 상태를 3단계로 나눠, 배포 단계 및 사용 여부를 명확히 함. 
+      - Staging: 테스트 중인 모델.
```
실험 조건에 맞는 상위 n개 모델. << 이 표현이 더 좋아보여요.

imella0707 replied:
네 조금 더 구체적으로 서술해주는 게  좋을 거 같네요. 감사합니다 !

dnlpys commented on README.md:
```diff
@@ -1 +1,198 @@
+    - 모델 상태 관리: 모델의 상태를 3단계로 나눠, 배포 단계 및 사용 여부를 명확히 함. 
+      - Staging: 테스트 중인 모델.
+      - Production: 실제 서비스 중인 모델, 배포 대상으로 지정됨. 
+      - Archived: Porduction에 있던 과거 모델은 archived로 변경, 더 이상 사용되지 않는 모델.
```
더 이상 사용하진 않는다는 표현은 적절하지 못해보입니다. 새로 배포된 모델의 성능에 문제가 있을 경우, 롤백해서 다시 사용해야될 경우가 있을수 있습니다.

archived로 변경. 까지만 쓰는것이 적절해 보입니다.

imella0707 replied:
네 롤백해야될 경우가 있으니까 더이상 사용하지 않는다고 단정지으면 안 될 것 같아요. 감사합니다 ~

dnlpys commented on README.md:
```diff
@@ -1 +1,198 @@
+
+### 프로메테우스
+- 데이터 수집 방식: pull 방식 선택, 모니터링 대상 각각의 서버마다 설치하지 않고 monitoring 서버에만 prometheus설치함. 
+- Logstash나 Telegraf같은 Pushing 방식보다 모니터링 서버에 영향이 가는 장애가 적은 편이어서 모니터링 도구로 선택하게 되었음. 
```
👍

imella0707 committed:
Update README.md

1.mlflow 서술 수정
-staging 
-archived 

2.프로메테우스 서술 수정

imella0707 committed:
chore(readme): 리드미 업데이트 -4 (#93)

1.mlflow 서술 수정
-staging 
-archived 

2.프로메테우스 서술 수정
3.commit 메시지 변경

imella0707 committed:
chore(readme): 리드미 업데이트 -5 (#93)

프로메테우스, 그라파나 part 문장 다듬음.

dnlpys reviewed (approved):
LGTM👍👍👍
